packages/sources/peregrine-fund-admin/src/endpoint/nav.ts

```diff
@@ -0,0 +1,51 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { httpTransport } from '../transport/nav'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+
+export const inputParameters = new InputParameters(
+  {
+    assetId: {
+      required: true,
+      type: 'string',
+      description: 'The identifying number for the requested asset',
+    },
+  },
+  [
+    {
+      assetId: '100',
+    },
+  ],
+)
+
+export interface ResponseSchema {
```

Comment:
Is this being used somewhere?

Reply:
Good catch, duplicate and redundant, removed.